ndn_python_repo/utils/concurrent_fetcher.py

```diff
@@ -87,10 +89,10 @@ async def _retry(seq: int):
                     # cancel the Interests for non-existing data
                     for task in aio.all_tasks():
                         task_name = task.get_name()
-                        task_num = None
                         try:
                             task_num = int(task_name)
-                        except: continue
+                        except: # fixme: what specific error does this catch?
```

Comment:
`aio.all_tasks` returns tasks not created with this module. They have internal names generated by Python.
BTW I don't think it is a good idea to use task names.